src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CFTTaskMapper.java

```diff
@@ -309,12 +305,6 @@ private void mapVariableToTaskResourceProperty(TaskResource taskResource, String
                     SecurityClassification sc = SecurityClassification.valueOf((String) value);
                     taskResource.setSecurityClassification(sc);
                     break;
-                case TASK_ID:
-                    taskResource.setTaskId((String) value);
-                    break;
-                case TASK_NAME:
-                    taskResource.setTaskName((String) value);
-                    break;
```

Comment:
Curious: What was the reasoning to remove these? 
I can see why the task Id can be removed since it is part of the skeleton but the task_name and case_id should remain.

Reply:
No test coverage for those fields and I guess these fields fill by task initiation. And all Ft's pass without them

Reply:
@raghera do you have any suggestion for removing fields?

Reply:
I would probably have to look through the details a bit here.  Looks like the TaskResource (which get's updated onto the DB) is getting updated by an outcome from the configuration process?  

In which case the question is what should be updatable from that process and need to do that separately.  I think the TaskId is probably not something we want to be overwritten so can remove that one.  

We are not really restricting anything at the moment, we let the service team decide that.  So on that principle we can keep the rest there.